lib/chef/knife/bootstrap/windows-chef-client-msi.erb

```diff
@@ -263,6 +263,11 @@ IF NOT !psexitcode!==0 (
   echo *******************************************************************
 <% unless @chef_config[:knife][:bootstrap_run_command] %>
   echo First Chef run exit code was not 0
+  echo Now dumping Chef stacktrace log
+  echo *******************************************************************
+  type c:\chef\cache 2> nul
```

Comment:
Should be ` type C:\chef\cache\chef-stacktrace.out` I think

Reply:
Fixed!